bin/vpn-dns

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+echo "Monitoring for VPN DNS changes"
+
+while true; do
+  output=$(scutil << EOF
+list "State:.*VPN"
+exit
+EOF
+  )
+
+  if [[ $output =~ State:/Network/Service/(.+)/VPN ]]; then
```

Comment:
I'm realizing that if multiple VPNs are connected this will only be able to 'see' the first one.

Reply:
I did a little testing and only wireguard VPNs set this key and the app will only allow one tunnel to be active at a time so I'm marking this as WONTFIX.